master/usage/routereflector/bird-rr-config.md

```diff
@@ -42,20 +42,60 @@ single `bird` package installs both IPv4 and IPv6 BIRD):
     sudo apt-get update
     sudo apt-get install bird
 
-#### RHEL 7
-
-First, install EPEL. Depending on your system, the following command may
-be sufficient:
-
-    sudo yum install epel-release
-
-If that fails, try the following instead:
-
-    sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-
-With that complete, you can now install BIRD:
-
-    yum install -y bird{,6}
+#### RHEL
+
+> **Note**: The following commands require root privileges. You can either open a root shell
+> or prefix them with `sudo`.
+{: .alert .alert-info}
+
+1. From a terminal prompt, create a new file called bird.repo in the 
+   `/etc/yum.repos.d/` directory.
+
+   ```bash 
+   vi /etc/yum.repos.d/bird.repo
+   ```
+
+1. Add the following lines to the file.
+
+   ```
+   [bird]
+   name=Network.CZ Repository
+   baseurl=ftp://repo.network.cz/pub/redhat/
+   enabled=1
+   gpgcheck=0
+   gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-network.cz
+   ```
+
+1. Save and close the file.
+
+1. If you don't already have the `/pki/rpm-gpg/` directory, use the following command
+   to create it.
+   
+   ```bash
+   mkdir /etc/pki/ /etc/pki/rpm-gpg/
```

Comment:
Common usage here would be `mkdir -p /etc/pki/rpm-gpg`.  (The `-p` means to create any parent directories that aren't already there.)